Visual Studio Integration in FilleStatusList breaks FormCommit designer

### Environment

- Git Extensions 33.33.33
- Build 79a312b929947bce9a3ce5f3e2096c11f543ad31
- Git 2.41.0.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.21
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.19 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.21 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

- Microsoft Visual Studio Community 2022 (64-bit) - Current Version 17.7.0


### Issue description

```
Build started: Project: GitUI, Configuration: Debug Any CPU ------
1>C:\git\gitextensions\GitUI\CommandsDialogs\FormCommit.Designer.cs(1461,32,1461,40): error CS0649: Field 'FormCommit.Unstaged' is never assigned to, and will always have its default value null
1>C:\git\gitextensions\GitUI\CommandsDialogs\FormCommit.Designer.cs(1474,32,1474,38): error CS0649: Field 'FormCommit.Staged' is never assigned to, and will always have its default value null
1>Done building project "GitUI.csproj" -- FAILED.
```

After opening FormCommit and making a change and then building, you get the above error message.  The VisualStudio Integration should only happen in runtime and not design time

### Steps to reproduce

1. Build
2. Open FormCommit Designer
3. Make a change like change commit button text and put back so no real change
4. Save
5. Build

### Did this work in previous version of GitExtensions?

_No response_

### Diagnostics

_No response_

--- a/GitUI/UserControls/FileStatusList.cs
+++ b/GitUI/UserControls/FileStatusList.cs
@@ -102,8 +102,6 @@ namespace GitUI
 
             base.Enter += FileStatusList_Enter;
 
-            VisualStudioIntegration.Init();
-
             return;
 
             ImageList CreateImageList()
@@ -209,6 +207,12 @@ namespace GitUI
                 };
                 return item;
             }
+        }
+
+        protected override void OnRuntimeLoad()
+        {
+            base.OnRuntimeLoad();
+            VisualStudioIntegration.Init();
         }
 
         public void Bind(Func<ObjectId, string> describeRevision, Func<GitRevision, GitRevision> getActualRevision)

